compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -1028,4 +1029,27 @@ std::optional<int> getGPUSubgroupSize(mlir::FunctionOpInterface func) {
   return std::nullopt;
 }
 
+void QueryMMAIntrinsics(mlir::ModuleOp moduleOp,
+                        SmallVector<IREE::GPU::MMAAttr> &mmaAttrs) {
```

Comment:
I think this should probably return a mapping of executable variants to their mma attrs. Should should also have a test of a module with two variants.